docs/install/tools.rst

```diff
@@ -0,0 +1,58 @@
+.. _install_tools:
+
+Router Table Check Tool
+=======================
+
+The router check tool checks whether the route parameters returned by a router match what is expected.
+The tool can also be used to check whether a path redirect, path rewrite, or host rewrite
+match what is expected.
+
+Input
+  The tool expects two input JSON files:
+
+  1. A router config JSON file. The router config JSON file schema is found :ref:`here <config_http_conn_man_route_table>`.
+
+  2. A tool config JSON file. The tool config JSON file schema is found :ref:`here <config_tools_router_check_tool>`.
+     The tool config input file specifies urls (composed of authorities and paths)
+     and expected route parameter values. Additonal parameters such as additonal headers are optional.
+
+Output
+  The program exits with status EXIT_FAILURE if any test case does not match the expected route parameter
+  value.
+
+  The ``--details`` option prints out details for each test case. The first field indicates
+  a P for a correct match and a F for a failed match. The second field is the expected route parameter value.
+  The third field is the actual route parameter value. The fourth field indicates the parameter that is
+  compared. For example: ::
+
+    P instant-server instant-server cluster_name
```

Comment:
I think the output for a successful test case should just print the test case name.